src/md/src/GettingStarted/ReportPortalJobs.md

```diff
@@ -1,37 +1,47 @@
 ## Report Portal jobs configuration
 
-Currently, Report Portal has 3 configurable jobs for data removing:
+Currently, Report Portal has 5 configurable jobs for data removing:
 * Clean launches
 * Clean logs
-* Clean attachments
+* Clean attachments info
+* Clean attachments binaries
+* Project binary storage size recalculation
 
-Job execution period can be configured using environment variables, where period value should be provided in accordance with one of the patterns (ISO 8601):
-* P1M - every month
-* P1D - every day
-* PT1H - every hour
-* PT1M - every minute
-* PT10H5M3S - 10 hours 5 minutes 3 seconds period
+Job execution period can be configured using environment variables, where period value should be provided as cron expression:
```

Comment:
As **spring** cron expression, may be we should add some link with explanation, for example https://www.baeldung.com/cron-syntax-linux-vs-spring, in order to avoid some mistakes - using unix cron format instead of spring cron.

Reply:
opted